### PR TITLE
Feature/v2

### PR DESCRIPTION
**Pull request by demilad**
NigeriaLogos V2:
Design update to the existing site.
- Improved category sorting
- Improved scrolling back to top
- Improved jump to alphabets

NOTE: 
- PR is pretty big so it's best if each commit is approached like a separate PR
- Mobile optimisations with the new designs have not yet been implemented and will be sent with the next PR

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -49,12 +49,39 @@
 
 <body>
     <header>
+        <div class="scroll-show">
+            <div class="white-bg"></div>
+            <div class="site-logo">
+                <a href="https://nigerialogos.com/"><img src="images/nigerialogos_logo.svg" alt="NigeriaLogos logo"></a>
+            </div>
+            <div class="secondary-alphabet">
+                <a href="##">#</a>
```
Comment (fathermerry) [inline index.html]: Is there a significance for the double pound?

Comment (Odusanya) [reply index.html]: @fathermerry Lol, I'm guessing because scroll to ID is `#`, so it's saying scroll to the div with id=#.

Comment (demilad) [reply index.html]: @fathermerry exactly what @Odusanya said

Comment (aeadedoyin) [reply index.html]: @demilad This is dope!

Comment (demilad) [reply index.html]: thanks man! @aeadedoyin

**Pull request merged**